.github/ISSUE_TEMPLATE/release-checklist.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: Release checklist
+about: The process for making a new release
+title: Release VERSION
+labels: ""
+assignees: ""
+---
+
+<!--
+NOTE TO RELEASE CAPTAINS:
+
+When filing this issue, replace VERSION and MILESTONE placeholders with the
+correct values in all fields.
+-->
+
+Welcome Release Captain ! ⛵️
+
+- [ ] Assign this issue to yourself
+- [ ] Check the [VERSION milestone](https://github.com/mozilla-extensions/nimbus-devtools/milestone/MILESTONE)
+  - Any open issues (except this one) should be moved into a future milestone.
+- [ ] Check out the main branch: `git checkout main`
+- [ ] Check the following files have the expected version:
+  - `package.json`
+  - `web-ext-config.json`
+
+  If they don't, update them in a separate PR.
+
+- [ ] Create a tag for this release:
+
+```sh
+git tag release/vVERSION # e.g., release/v0.1.0
+git push origin --tags
+```
+
+- [ ] File a 
```

Comment:
Just wondering, is this supposed to be incomplete?

Reply:
Indeed, it was not :)